[WIP] Alt text

CleezyITP opened the pull request:
I think I am still adding in the alt text improperly....

<img width="1185" alt="screen shot 2016-08-20 at 5 06 07 pm" src="https://cloud.githubusercontent.com/assets/5375410/17833893/b5160a1a-66f8-11e6-8a34-ea011d58cd9f.png">

Can you look at where I am adding the @alt and let me know where they should be?

lmccart commented on src/color/setting.js:
```diff
@@ -102,6 +102,48 @@ require('./p5.Color');
  * </code>
  * </div>
+ *
+ ** @alt
```
this should be a single asterisk here

lmccart replied:
I've made the change already, you can just pull the newest

lmccart commented:
I didn't get a chance to go through this line by line yet, but I looked at a few (color, noLoop, background) and it all looks like you're doing it right. I'm not sure why it's not showing up correctly for you when you build. I can look into it more later. 

For now, I posted what you had submitted so far to p5js.org, so you can test that at least.
http://p5js.org/reference/#p5/background

lmccart commented on src/core/curves.js:
```diff
@@ -63,6 +63,16 @@ var curveDetail = 20;
  *bezier(250,250,0, 100,100,0, 100,0,0, 0,100,0);
  * </code>
  * </div>
+ *@alt
```
this one actually needs to be attached to the first bezier block above. i've moved it here:
https://github.com/processing/p5.js/commit/80f003001ec1304dbf49713cc68ad8a86fe143d4

lmccart commented:
Ok two things I noticed:

1. I found alts were not showing up for bezier because the @alt block was actually attached to the second definition block of two for bezier. I've moved it up. This is an unfortunately finicky and easy to mixup part, I hope it's not too much of a hassle for you. I guess the key is whenever you insert some @alt block, look above and check that there is no other definition for that function.

2. I've realized that a lot of these lines are over the 80chars limit, so we'll have to come up with some other delimiter besides a new line. This shouldn't be a problem to switch in code and I can do a find and replace to switch what you've done already. I'll let you know when I've updated things, but for now, you can continue using line breaks.

CleezyITP commented:
Oop! All alts are 80 characters or less. I just lumped them all together in
one block (one line= one alt). I think I have been doing it all wrong...Now
disbursing alts more

On Sun, Aug 21, 2016 at 2:59 PM, Lauren McCarthy <notifications@github.com>
wrote:

> Ok two things I noticed:
>
>    1.
>
>    I found alts were not showing up for bezier because the @alt
>    <https://github.com/alt> block was actually attached to the second
>    definition block of two for bezier. I've moved it up. This is an
>    unfortunately finicky and easy to mixup part, I hope it's not too much of a
>    hassle for you. I guess the key is whenever you insert some @alt
>    <https://github.com/alt> block, look above and check that there is no
>    other definition for that function.
>    2.
>
>    I've realized that a lot of these lines are over the 80chars limit, so
>    we'll have to come up with some other delimiter besides a new line. This
>    shouldn't be a problem to switch in code and I can do a find and replace to
>    switch what you've done already. I'll let you know when I've updated
>    things, but for now, you can continue using line breaks.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/processing/p5.js/pull/1562#issuecomment-241275022>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFIFsmUc8IBLlxgoMz-RDQp5SzhhIaXkks5qiKAbgaJpZM4JpKjp>
> .
>



-- 
Claire Kearney-Volpe
MPS, ITP (NYU)

takinglifeseriously.com
